src/components/SecondaryPanes/Expressions.js

```diff
@@ -174,18 +178,30 @@ class Expressions extends PureComponent<Props, State> {
         return;
       }
 
+      e.persist();
```

Comment:
? never seen this

Reply:
this allows references to the event to be retained, without it the event becomes null after calling hasSyntaxtError